spec/features/user_views_activity_feed_spec.rb

```diff
@@ -34,4 +34,24 @@
                                     assignment: other_department_result.assignment.name,
                                     subject: other_department_result.assignment.subject)
   end
+
+  scenario "sees her username is associated with each activity" do
+    outcome = create(:outcome)
+    course = outcome.course
+    coverage = create(:coverage, course: course, outcomes: [outcome])
+    user = user_with_assignments_access_to(course.department)
+    user_creates_an_assignment(user, course)
+
+    visit activities_path(as: user)
+
+    expect(page).to have_content user.username
+  end
+
+  def user_creates_an_assignment(user, course)
+    visit manage_assignments_course_path(course.id, as: user)
```

Comment:
can we speed this up a tad by going right to `new_manage_assignments_outcome_coverage_assignment_path` (wow, that's a mouthful) rather than navigating to the course first?